docs/current_docs/cloud/572923-get-started.mdx

```diff
@@ -36,58 +36,65 @@ At the end of this step, you will have signed up for Dagger Cloud and obtained a
 
 Follow the steps below to sign up for Dagger Cloud, create an organization and obtain a Dagger Cloud token.
 
-1. Browse to the [Dagger Cloud website](https://dagger.cloud/signup). Click *Continue with GitHub* to log in with your GitHub account.
-
-    ![Authenticate with GitHub](/img/current_docs/cloud/get-started/connect-github.png)
+1. Sign up for Dagger Cloud by selecting a plan on the [Dagger website](https://www.dagger.io/pricing). Click *Continue with GitHub* to log in with your GitHub account.
 
-1. On the GitHub authorization screen, confirm the Dagger Cloud connection for authentication. Once authorized, you will be redirected to a welcome page and prompted to create a new Dagger Cloud organization. Enter a name for your organization in the *Organization Name* field. Click *Next* to proceed.
+1. After authorizing Dagger Cloud for your GitHub account, you'll create your organization.
 
-    :::info
-    The organization name can only contain alphanumeric characters and dashes and is unique across Dagger Cloud.
+    :::info Naming your organization
+    Organization names may contain alphanumeric characters and dashes and are unique across Dagger Cloud. We recommend using your company name or team name for your organization.
     :::
 
-    ![Create Dagger Cloud organization](/img/current_docs/cloud/get-started/create-organization.png)
-
 1. Review the available Dagger Cloud subscription plans. Choose a plan by clicking *Select*.
-1. If you selected the *Team* plan, you will be presented with the option to add teammates to your Dagger Cloud account. This step is optional and not available in individual plans. Enter one or more email addresses as required. Click *Next* to proceed.
 
-    ![Select plan](/img/current_docs/cloud/get-started/invite-team.png)
+1. If you selected the *Team* plan:
```

Comment:
I don't think `*Team*` actually makes it bold since in the page you can barely notice the difference :thinking: 
![image](https://github.com/dagger/dagger/assets/1578458/475c8374-f574-4777-ab95-c891e96b892c)

Reply:
Yeah, that's making them italicized. I'm updating them to **text** in a commit coming in soon.